presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotPushdownUtils.java

```diff
@@ -147,10 +154,25 @@ public ExpressionType getExpressionType()
             if (agg != null) {
                 if (agg.getFilter().isPresent()
                         || agg.isDistinct()
-                        || agg.getOrderBy().isPresent()
-                        || agg.getMask().isPresent()) {
+                        || agg.getOrderBy().isPresent()) {
                     throw new PinotException(PINOT_UNSUPPORTED_EXPRESSION, Optional.empty(), "Unsupported aggregation node " + aggregationNode);
                 }
+                if (agg.getMask().isPresent()) {
+                    /**
```

Comment:
We don't use javadoc in the body of a function use comments instead
```java
// ...
// ...
// ...

Reply:
done